test/helpers.ts

```diff
@@ -279,3 +280,31 @@ export function prettyPrintPuppeteerAssertionError(message) {
 		assert.equal(match[1], match[2]);
 	}
 }
+
+export async function getNewPage(browser: Browser): Promise<Page> {
+	const pages = await browser.pages();
+	if (pages.length) return pages[0];
```

Comment:
Shouldn't we intentionally create a new page, instead of sharing it, to potentially avoid state leaking between tests? I think the slower test time may still be worth it to ensure correctness.

Reply:
You are correct.

I choose this way because `EXC_BAD_ACCESS (SIGSEGV)` is occurred mainly when creating a new page.
(And I thought that Svelte doesn't depend on cookies, SessionStrage, LocalStrage, so this isn't the real problem.😅)

But now again I checked here. And possibly the cause of `EXC_BAD_ACCESS (SIGSEGV)` is too many tabs.
I don't see `EXC_BAD_ACCESS (SIGSEGV)` after add process that deleting pages.

Please check this commit! https://github.com/sveltejs/svelte/pull/7076/commits/8eb13005ac4cfec17865a4b62f2bab088682a226

Reply:
Nice! Yeah it definitely seems like the tests didn't close the page. Great that you added that